packages/expo-updates/ios/EXUpdates/UpdatesStateMachine.swift

```diff
@@ -160,6 +165,16 @@ internal struct UpdatesStateContext {
   let downloadedManifest: [String: Any]?
   let checkError: [String: String]?
   let downloadError: [String: String]?
+  let lastCheckForUpdateTime: Date?
+
+  private var dateString: String? {
+    switch lastCheckForUpdateTime {
+    case nil:
+      return nil
+    default:
+      return iso8601DateFormatter.string(from: lastCheckForUpdateTime ?? Date())
+    }
+  }
```

Comment:
can use guard statement for type narrowing. also let's name this according to the field it is stringifying

```suggestion
  private var lastCheckForUpdateTimeDateString: String? {
    guard let lastCheckForUpdateTime = lastCheckForUpdateTime else {
      return nil
    }
    return iso8601DateFormatter.string(from: lastCheckForUpdateTime)
  }
```

Reply:
oops I thought I had made that change already, thanks

Reply:
Done